website/docs/Debugging.md

```diff
@@ -127,3 +127,37 @@ If you are an [Atom](https://atom.io/) hacker you can try [`wdio-repl`](https://
 You can create a node.js debug configuration like this:
 ![Screenshot from 2021-05-29 17-33-33](https://user-images.githubusercontent.com/18728354/120088460-81844c00-c0a5-11eb-916b-50f21c8472a8.png)
 Watch this [YouTube Video](https://www.youtube.com/watch?v=Qcqnmle6Wu8) for more information about how to make a configuration.
+
+## Debugging flaky tests
+
+Flaky tests can be really hard to debug so here are some tips how you can try and get that flaky result you got in your CI, reproduced locally.
+
+### Network
+To debug network related flakiness use the [throttleNetwork](https://webdriver.io/docs/api/browser/throttleNetwork) command.
+```js
+await browser.throttleNetwork('Regular3G')
+```
+
+### Rendering speed
+To debug device speed related flakiness use the [throttleCPU](https://webdriver.io/docs/api/browser/throttleCPU) command.
+This will cause your pages to render slower which can be caused by many things like running multiple processes in your CI which could be slowing down your tests.
+```js
+await browser.throttleCPU(4)
+```
+
+### Test execution speed
+If your tests do not seem to be affected it is possible that WebdriverIO is faster than the update from the frontend framework / browser.
+This happens when using synchronous assertions, some examples of code that can break because of this:
```

Comment:
```suggestion

If your tests do not seem to be affected it is possible that WebdriverIO is faster than the update from the frontend framework / browser. This happens when using synchronous assertions since WebdriverIO has no chance to retry these assertions anymore. Some examples of code that can break because of this:
```